src/Configuration/CFGCommon/CFGCommon.cpp

```diff
@@ -422,10 +422,47 @@ int CFG_execute_cmd(const std::string& cmd, std::string& output) {
   return exitcode;
 }
 
+int CFG_execute_cmd(const std::string& cmd, std::string& output,
```

Comment:
Why this function is needed? 

Is this true that we can just call CFG_execute_cmd_with_callback() but with both callback set to null? 

We can achieve the same result right?

Reply:
You are right. I will clean it up